src/esp/physics/RigidObject.cpp

```diff
@@ -213,5 +213,35 @@ Magnum::Matrix3 RigidObject::getInertiaMatrix() {
   return inertia;
 }
 
+//////////////////
+// VelocityControl
+Magnum::Matrix4 VelocityControl::integrateTransform(
```

Comment:
@mosra: have you read this functions? 
Any advanced matrix manipulation from Magnum library can kick in to improve it?

Reply:
[See my earlier comments.](https://github.com/facebookresearch/habitat-sim/pull/486#discussion_r378325455)